vncviewer/DesktopWindow.cxx

```diff
@@ -875,50 +873,62 @@ int DesktopWindow::fltkHandle(int event, Fl_Window *win)
   return ret;
 }
 
-
 void DesktopWindow::fullscreen_on()
 {
-  if (not fullScreenAllMonitors)
-    fullscreen_screens(-1, -1, -1, -1);
-  else {
-    int top, bottom, left, right;
-    int top_y, bottom_y, left_x, right_x;
+  int top, bottom, left, right;
+  top = bottom = left = right = -1;
+  bool all_monitors = !strcasecmp(fullScreenMode, "all");
 
-    int sx, sy, sw, sh;
+  if (all_monitors) {
+      std::set<int> monitors;
```

Comment:
Incorrect indentation here, which is also making this diff more difficult to read then it has to be.